app/models/maintenance_stats/stats/bitbucket/commit_count_stats.rb

```diff
@@ -0,0 +1,21 @@
+module MaintenanceStats
+    module Stats
+        module Bitbucket
+            class CommitsStat < MaintenanceStats::Stats::BaseStat
+                def count_up_commits(dataset, since)
+                    return nil if dataset.nil? || dataset.empty?
+                    dataset.select {|commit| Date.parse(commit["date"]) >= since}.size
+                end
+
+                def get_stats
+                    {
+                        "last_week_commits": count_up_commits(@results, 1.week.ago),
+                        "last_month_commits": count_up_commits(@results, 1.month.ago),
```

Comment:
Do we want to reference these all from the same time? It's possible to roll over a day or whatever and get unexpected results